dotcom-rendering/src/web/components/Card/components/LI.tsx

```diff
@@ -85,19 +101,22 @@ export const LI = ({
 	padBottomOnMobile,
 	showTopMarginWhenStacked,
 	snapAlignStart = false,
+	offsetBottomPaddingOnDivider = false,
 }: Props) => {
 	// Decide sizing
 	const sizeStyles = decideSize(percentage, stretch);
+	const paddingSize = '10px';
```

Comment:
I think it could worth adding comment here describing _why_ we're defining paddingSize in a central place for this component?